WBVALVE/WBVALVE-IOC-01App/Db/Makefile

```diff
@@ -0,0 +1,22 @@
+TOP=../..
```

Comment:
Since the DB is in the support module rather than the IOC you don't need this directory.